css/styles.css

```
@@ -0,0 +1,508 @@
+.category__image-5 {
+	background: url(/img/cat-smartwatch-logo.svg) #ffe17f 50% 50% no-repeat;
+}
+.category__image-6 {
+	background: url(/img/cat-quadro-logo.svg) #ffe17f 50% 50% no-repeat;
+}
+
+.category__content {
+	font-family: "Raleway", sans-serif;
+	font-size: 18px;
+	line-height: 24px;
+}
+
+/* свойства секции "services" */
+
+.service__tab {
+	background-color: #f0f0f0;
+}
+
+.tab-label {
+	display: block;
+	width: 160px;
+	height: 20px;
+	font-family: "Raleway", sans-serif;
+	font-size: 18px;
+	line-height: 24px;
+	background-color: #ffe17f;
+	text-transform: uppercase;
+}
+
+.tab-label-current {
+	width: 280px;
+	height: 40px;
+	background-color: #000000;
+	color: #ffe17f;
+}
+
+.tab-description {
+	font-family: "Raleway", sans-serif;
+	font-size: 50px;
+	line-height: 50px;
+}
+
+/* свойства секции "about" */
+
+.about__delivery {
+	background-color: #f0f0f0;
+}
+
+.delivery__image {
+	width: 125px;
+	height: 100px;
+	background: url(/img/delivery-logo.svg) #ffe17f 50% 50% no-repeat;
+}
+
+.delivery__content {
+	font-family: "Raleway", sans-serif;
+	font-size: 20px;
+	line-height: 24px;
+	text-transform: uppercase;
+	letter-spacing: 0.2em;
+}
+
+.delivery__button {
+	width: 40px;
+	height: 40px;
+	background: url(/img/delivery-info-logo.svg) #dcdcdc 50% 50% no-repeat;
+	border: none;
+	border-radius: 50%;
+	cursor: pointer;
+}
+
+.info__title,
+.contact__title {
+	font-family: "Raleway", sans-serif;
+	font-size: 50px;
+	line-height: 50px;
+}
+
+.info__list-item {
+	font-weight: 700;
+	line-height: 20px;
+}
+
+.contact__subtitle {
+	font-size: 24px;
+	line-height: 30px;
+}
+
+.contact__content {
+	font-style: normal;
+	color: #000000;
+}
+
+/* свойства секции "subscribe" */
+
+.subscribe {
+	background-color: #f0f0f0;
+}
+
+.subscribe__slogan {
+	font-family: "Raleway", sans-serif;
+	font-size: 50px;
+	line-height: 50px;
+}
+
+.subscribe__input {
+	color: rgba(68, 68, 68, 0.5);
+	border: none;
+	border-bottom: 3px solid rgba(0, 0, 0, 0.1);
+	background-color: inherit;
+}
+
+.subscribe__button {
+	font-family: "Raleway", sans-serif;
+	color: #000000;
+	letter-spacing: 0.2em;
+	text-transform: uppercase;
+	border: 3px solid rgba(0, 0, 0, 0.1);
+	background-color: none;
+	cursor: pointer;
+}
+.subscribe__content {
+	color: #444444;
+}
+
+.subscribe__check {
+	width: 14px;
+	height: 14px;
+	background: url(/img/subscribe-check.svg) 50% 50% no-repeat;
+}
+
+/* свойства "footer" */
+
+.page-footer {
+	background-color: #444444;
+	color: #ffffff;
+}
+
+.page-footer__address-text,
+.page-footer__contact {
+	font-style: normal;
+	text-decoration: none;
+	color: #ffffff;
+}
+
+.socials__link {
+	display: block;
+	width: 40px;
+	height: 40px;
+	border-radius: 50%;
+}
+
+.socials__link-1 {
+	background: url(/img/facebook-logo.svg) #ffffff 50% 50% no-repeat;
+}
+
+.socials__link-2 {
+	background: url(/img/instagram-logo.svg) #ffffff 50% 50% no-repeat;
+}
+
+.socials__link-3 {
+	background: url(/img/twitter-logo.svg) #ffffff 50% 50% no-repeat;
+}
```

Я бы предложил назвать классы по другому
```suggestion
.socials__link-facebook {
	background: url(/img/facebook-logo.svg) #ffffff 50% 50% no-repeat;
}

.socials__link-instagram {
	background: url(/img/instagram-logo.svg) #ffffff 50% 50% no-repeat;
}

.socials__link-twitter {
	background: url(/img/twitter-logo.svg) #ffffff 50% 50% no-repeat;
}
```

---

Ок, изменю названия